integrations/tensorflow/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1549,10 +1546,10 @@ static SmallVector<OpFoldResult> getMixedTiles(OpTy op) {
                 "applies to only pack or unpack operations");
   SmallVector<OpFoldResult> mixedInnerTiles;
   unsigned dynamicValIndex = 0;
-  for (Attribute attr : op.getStaticInnerTiles()) {
-    auto tileAttr = attr.cast<IntegerAttr>();
-    if (!ShapedType::isDynamic(tileAttr.getInt()))
-      mixedInnerTiles.push_back(tileAttr);
+  OpBuilder b(op.getContext());
```

Comment:
Using builders this way I have always seen as a red flag. Let me see if I can drop this

Reply:
it can be removed, the syntax is just a bit more verbose.

Reply:
Fair enough.